A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -86,6 +94,108 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this.electricity = this.querySelector("#Electricity");
         this.climbTransitionGroundAltitude = null;
     }
+
+    insertSmallFontSpan() {
+        this._lineElements.forEach((row) => {
+            row.forEach((column) => {
+                if (column != null) {
+                    column.innerHTML = column.innerHTML.replace(/{smallFront}/g, "<span class='s-text'>");
+                    column.innerHTML = column.innerHTML.replace(/{smallEnd}/g, "</span>");
+                }
+            })
+        })
+    }
+
+    /**
+     * Need to refactor this absolute slop
+     * @param s
+     * @returns {boolean|Promise<boolean>}
+     */
+    tryParseAverageWind(s) {
+        let wind = 0;
+        if (s.includes("HD")) {
+            wind = parseFloat(s.split("HD")[1])
+            this._windDir = "HD"
+            if (isFinite(wind)) return this.trySetAverageWind(wind)
+            else {
+                this.showErrorMessage("FORMAT ERROR")
+                return false
+            }
+        } else if (s.includes("H")) {
+            wind = parseFloat(s.split("H")[1])
+            this._windDir = "HD"
+            if (isFinite(wind)) return this.trySetAverageWind(wind)
+            else {
+                this.showErrorMessage("FORMAT ERROR")
+                return false
+            }
+        } else if (s.includes("-")) {
+            wind = parseFloat(s.split("-")[1])
+            this._windDir = "HD"
+            if (isFinite(wind)) return this.trySetAverageWind(wind)
+            else {
+                this.showErrorMessage("FORMAT ERROR")
+                return false
+            }
+        } else if (s.includes("TL")) {
+            wind = parseFloat(s.split("TL")[1])
+            this._windDir = "TL"
+            if (isFinite(wind)) return this.trySetAverageWind(wind)
+            else {
+                this.showErrorMessage("FORMAT ERROR")
+                return false
+            }
+        } else if (s.includes("T")) {
+            wind = parseFloat(s.split("T")[1])
+            this._windDir = "TL"
+            if (isFinite(wind)) return this.trySetAverageWind(wind)
+            else {
+                this.showErrorMessage("FORMAT ERROR")
+                return false
+            }
+        } else { // Until the +- button the MCDU actually shows a plus sign
+            wind = parseFloat(s)
+            this._windDir = "TL"
+            if (isFinite(wind)) return this.trySetAverageWind(wind)
+            else {
+                this.showErrorMessage("FORMAT ERROR")
+                return false
+            }
+        }
+    }
+
+    tryParseZeroFuelWeightZFWCG(s) {
```

Comment:
You can modify the `trySetZeroFuelWeightZFWCG` function directly if you include the needed file, https://github.com/flybywiresim/a32nx/pull/871/commits/a83295cabfec06054a220166378a8377f324a0f3 is exactly that, you can copy that commit to this PR.
Same goes for the custom `trySetAverageWind()` as well.

If you need to modify something that's in a shared file, follow the procedure from that commit to add the file to the mod.